src/main/webapp/view.js

```diff
@@ -0,0 +1,15 @@
+/* Number of characters until lecture ID in URL. */
+const URL_ID_SPACING = 5;
```

Comment:
Where does the 5 come from? `?key=`?

Reply:
I think we should just use URL and URLSearchParams to do this parsing/generating

Reply:
i agree, ill make the change